src/WebJobs.Script.WebHost/Features/FunctionExecutionFeature.cs

```diff
@@ -62,10 +62,18 @@ public async Task ExecuteAsync(HttpRequest request, CancellationToken cancellati
             }
 
             var sw = Stopwatch.StartNew();
-            var arguments = new Dictionary<string, object>()
+
+            var arguments = new Dictionary<string, object>();
+            if (_descriptor.IsWarmupFunction())
+            {
+                ParameterDescriptor inputParameter = _descriptor.Parameters.First(p => p.IsTrigger);
```

Comment:
`_descriptor.TriggerParameter.Name` like we use below doesn't work for warmup functions?

Reply:
_descriptor.TriggerParameter.Name is not the issue here. it's the type.
basically this line adds HttpRequest object but we need the WamupContext object here

Reply:
My point was that it seems to get the name you can just use `_descriptor.TriggerParameter.Name` since that is the cached value you're re-looking up it seems.

Reply:
oh yes, updated it, thanks